README.md

```diff
@@ -1016,10 +1285,17 @@ const closeConnection = conn => Future((rej, res) => {
 ```
 
 #### finally
-##### `#finally :: Future a b ~> Future a c -> Future a b`
-##### `#lastly :: Future a b ~> Future a c -> Future a b`
-##### `.finally :: Future a c -> Future a b -> Future a b`
-##### `.lastly :: Future a c -> Future a b -> Future a b`
+
+<details><summary><code>finally :: Future a b ~> Future a c -> Future a b</code></summary>
+
+```hs
+finally :: Future a b ~> Future a c               -> Future a b
+lastly  :: Future a b ~> Future a c               -> Future a b
+finally ::               Future a c -> Future a b -> Future a b
+lastly  ::               Future a c -> Future a b -> Future a b
```

Comment:
I find it confusing for an expression to have different meanings depending on the type signatures associated with it. I would find `Future.finally`/`Future.prototype.finally` much clearer than `finally`/`finally`.